index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Chocolate Pizza | Delicious</title>
+        <meta name="Learn how to make a Chocolate Pizza dessert with this easy to follow recipe, complete with ingredients list." content="">
+        <link rel="stylesheet" href="css/reset.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link href='http://fonts.googleapis.com/css?family=Droid+Serif:400,700' rel='stylesheet' type='text/css'>
+    </head>
+
+    <body>
+      <header>
+        <img src="images/logo.jpg" class="logo">
+          <section class="contact">
+            <img src="images/facebookIcon.jpg" alt="Facebook">
+            <img src="images/twitterIcon.jpg" alt="Twitter">
+            <img src="images/googleIcon.jpg" alt="Google+">
+            <img src="images/instagramIcon.jpg" alt="Instagram">
+            <img src="images/flickrIcon.jpg" alt="Flickr">
+            <img src="images/pinterestIcon.jpg" alt="Pinterest">
+            <img src="images/rssIcon.jpg" alt="RSS">
+            <img src="images/emailIcon.jpg" alt="Email">
+          </section>
+        <img src="images/lineBreak.jpg" class="lineBreak">
+      </header>
+
+      <article>
+        <h1>Chocolate Pizza</h1>
+        <section class="title">
+          <span class="dateTag">Posted on 15 Dec 2013 / Desserts</span>
+          <img src="images/print.jpg" class="print">
+        </section>
+        <img src="images/mainImage.jpg">
+
+        <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p>
+
+        <p>Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+
+        <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+
+        <section class="checklist">
+          <img src="images/list.png">
+
+        </section>
+
+        <section class="authorSection">
+          <img src="images/lineBreak.jpg" class="lineBreak">
+          <section class="authorDetails">
+            <img src="images/author.jpg" class="authorImage">
+            <span class="authorName">Vanessa Stevenson</span><br />
```

Comment:
We don't use the br tag any longer. Adding additional line breaks should be done in the form of using CSS. You could avoid using having to do a line break here if you used a blog level element as opposed to the span tag which is displayed inline.